README.md

```diff
@@ -10,47 +10,70 @@
 [![Translate](https://img.shields.io/badge/translate-zanata-blue.svg)](https://translate.zanata.org/zanata/project/view/manageiq-ui-self_service)
 [![License](http://img.shields.io/badge/license-APACHE2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
 
-The Self Service UI for the [ManageIQ](http://github.com/ManageIQ/manageiq) project
+The Service UI (SUI) for the [ManageIQ](http://github.com/ManageIQ/manageiq) project.
 
-## Installation
+## Developer Setup
+Logging in to the SUI requires a running instance of ManageIQ. Instructions on how to install ManageIQ can be found 
+[here](https://github.com/ManageIQ/guides/blob/master/developer_setup.md).
 
-### Prerequisites
+### Install Repository and Prerequisites
 
-- Install nodejs 0.10.46 and npm 3.10.7
-- Install bower and gulp globally: `npm install -g bower gulp`
 - Have the [ManageIQ](http://github.com/ManageIQ/manageiq) repo cloned into a
   directory named `manageiq`, and ready to be started.
+- Have this repo as a sibling to the `manageiq` directory:
+`git clone git@github.com:ManageIQ/manageiq-ui-self_service.git`.
+- Have nodejs **0.10.46** and npm **3.10.7** installed ([NVM](https://github.com/creationix/nvm) is a popular solution 
+to manage node and npm versions).
+- Have Ruby version **2.2.2+** and the latest Bundler installed.
+  - `gem install bundler`
+- Have bower and gulp globally installed.
+  - `npm install -g bower gulp`
 
-### Repository and dependencies
+### Install Dependencies
 
-- Clone this repo as a sibling to the `manageiq` directory:
-  `git clone git@github.com:ManageIQ/manageiq-ui-self_service.git`
 - `cd manageiq-ui-self_service`
 - `bundle install`
```

Comment:
@AllenBW yeah, as a part of the MIQ install, "bundle install" doesn't need to be run per se (but maybe this is good in case someone doesn't have it installed).